html/js/ghShared.js

```diff
@@ -1989,3 +1991,18 @@ function setCookie(cName, value, expireDays) {
     document.cookie=cName + "=" + escape(value)+((expireDays==null) ? "" : ";expires="+exdate.toUTCString()) + ";path=/";
     document.cookie=cName + "=" + escape(value)+((expireDays==null) ? "" : ";expires="+exdate.toUTCString()) + ";path=/resourceType.py";
 }
+/* Resource Stat Copy */
+function addResourceCopy(resName) {
+    var resName = resName;
+    //var resName = $(".resAttr").data("resource");
+		var copySource = $("[data-resource='"+resName+"']").data("stats");
```

Comment:
Please fix indentation on this line.
```suggestion
    var copySource = $("[data-resource='"+resName+"']").data("stats");
```